PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/slimui/creation/WebAppCreationDialog.java

```diff
@@ -427,8 +432,10 @@ private void createWebApp() {
         ProgressManager.getInstance().run(new Task.Modal(null, "Creating New WebApp...", true) {
             @Override
             public void run(ProgressIndicator progressIndicator) {
-                try {
+                Map<String, String> properties = webAppConfiguration.getModel().getTelemetryProperties(null);
+                EventUtil.executeWithLog(WEBAPP, CREATE_WEBAPP, properties, null, (operation) -> {
```

Comment:
If event name is specified,  we could know the event type, we may simplify this later